transformers/test_data/dog_deny.go

```diff
@@ -0,0 +1,54 @@
+package test_data
+
+import (
+	"math/rand"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+)
+
+var (
+	RawDogDenyLog = types.Log{
+		Address: common.HexToAddress(Dog1xxAddress()),
+		Topics: []common.Hash{
+			common.HexToHash(constants.DogDenySignature()),
+			common.HexToHash("0x000000000000000000000000dDb108893104dE4E1C6d0E47c42237dB4E617ACc"), //usr
+		},
+		Data:        hexutil.MustDecode("0x"),
+		BlockNumber: testBlockNumber,
+		TxHash:      common.Hash{},
+		TxIndex:     0,
+		BlockHash:   common.Hash{},
+		Index:       0,
+		Removed:     false,
+	}
+
+	DogDenyEventLog = core.EventLog{
+		ID:          int64(rand.Int31()),
+		HeaderID:    int64(rand.Int31()),
+		Log:         RawDogDenyLog,
+		Transformed: false,
+	}
+	dogDenyModel = event.InsertionModel{
+		SchemaName: constants.MakerSchema,
+		TableName:  constants.DogDenyTable,
+		OrderedColumns: []event.ColumnName{
+			event.HeaderFK,
+			event.LogFK,
+			event.AddressFK,
+			constants.UsrColumn,
+		},
+		ColumnValues: event.ColumnValues{
+			event.HeaderFK: DogDenyEventLog.HeaderID,
+			event.LogFK:    DogDenyEventLog.ID,
+			//event.AddressFK,
```

Comment:
What's up with these? Not known until we have better test data?

Reply:
No, we don't know them since they are database values (addressID) so we don't know them until the test runs.